src/core/game-sound.js

```diff
@@ -23,12 +23,10 @@ const isAlreadyPlaying = audioKey => {
     return audioKey && Assets.backgroundMusic && audioKey === Assets.backgroundMusic.key;
 };
 
-const onFadeComplete = scene => {
-    Assets.backgroundMusic.destroy();
-    startNextMusic(scene);
-};
-
 const startNextMusic = scene => {
+    if (Assets.backgroundMusic) {
```

Comment:
Can maybe do this now: **Assets.backgroundMusic?.destroy()**

I think that works...

Reply:
Nice! Will try it.